Dockerfile-code

```diff
@@ -0,0 +1,12 @@
+FROM study-manager-base:latest
```

Comment:
Multistage builds in docker are pretty great--let one Dockerfile manage stuff that rarely changes, and then depend on it as infrastructure for a different Dockerfile.